support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -176,6 +176,7 @@ const discountSummaryCopy = (
 ) => {
 	/**
 	 * EXAMPLE:
+	 * - £6/month for the first month, then £10/month
```

Comment:
It might be nice to convert these scenarios to Jest tests if they aren't already.

Reply:
Can do a follow up for this, think need to get offer out.